src/config/AppConstant.js

```diff
@@ -2,11 +2,15 @@ const API_BASE_URL = import.meta.env.DEV ? "http://localhost:8000/" : "https://a
 const CREATE_ACCOUNT_URL = "signup/";
 const LOGIN_ACCOUNT_URL = "login/";
 const LOGOUT_ACCOUNT_URL = "logout";
+const COLLECTIONS_URL = "vault_collections/";
+const ITEM_URL = "vault_items/"
```

Comment:
What's the reason we make Collections plural but Item singular?

Reply:
It would make the URLS long, but I would consider adding VAULT as the first word of the URLs so that we can easily find all the vault related URLs.

Reply:
I think I was doing this aimlessly, I will fix this with the routes PR for next week just so it doesn't cause merge conflicts with Derth's next merge here. I'll add a note to fix this and the alphabetically order on export! 

* https://github.com/secure-password-manager/PasswordManagerWeb/pull/53/files